sdcm/tester.py

```diff
@@ -1276,7 +1276,7 @@ def alter_table_to_in_memory(self, key_space_name="keyspace1", table_name="stand
         compaction_strategy = "%s" % {"class": "InMemoryCompactionStrategy"}
         cql_cmd = "ALTER table {key_space_name}.{table_name} " \
                   "WITH in_memory=true AND compaction={compaction_strategy}".format(**locals())
-        node.remoter.run('cqlsh -e "{}" {}'.format(cql_cmd, node.private_ip_address), verbose=True)
+        self.db_cluster.run_cqlsh(node=node, cql_cmd=cql_cmd)
```

Comment:
is that exist in branch-2019.1 ? i.e. run_cqlsh ?

Reply:
Yes